hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconConstants.java

```diff
@@ -58,4 +58,9 @@ private ReconConstants() {
       + PIPELINE_DB_SUFFIX;
   public static final String RECON_SCM_NODE_DB =
       "recon-node.db";
+  public static final long MAX_FILE_SIZE_UPPER_BOUND = 1125899906842624L;
+  public static final long MIN_FILE_SIZE_UPPER_BOUND = 1024L;
```

Comment:
nvm. should be upper bound.

Reply:
No. I think it's still upper bound, but the minimum upper bound. Let's say the file size is 10L, which will be rounded to 1024L instead of 16L since 1024L is the minimum upper bound for file size.